lua/nvim-tree/actions/node/utils.lua

```diff
@@ -0,0 +1,44 @@
+-- Copyright 2019 Yazdani Kiyan under MIT License
+local notify = require("nvim-tree.notify")
+
+local M = {}
+
+---@alias ApiNodeDeleteWipeBufferMode '"delete"'|'"wipe"'
+
+---@param mode ApiNodeDeleteWipeBufferMode
+---@param filename string
+---@param opts ApiNodeDeleteWipeBufferOpts|nil
+---@return nil
+function M.delete_buffer(mode, filename, opts)
+  if type(mode) ~= "string" then
+    mode = "delete"
+  end
+
+  local buf_fn = vim.cmd.bdelete
+  if mode == "wipe" then
+    buf_fn = vim.cmd.bwipe
+  end
+
+  opts = opts or { force = false }
+
+  local notify_node = notify.render_path(filename)
+
+  -- check if buffer for file at cursor exists and if it is loaded
+  local bufnr_at_filename = vim.fn.bufnr(filename)
+  if bufnr_at_filename == -1 or vim.fn.getbufinfo(bufnr_at_filename)[1].loaded == 0 then
+    notify.error("No loaded buffer coincides with " .. notify_node)
```

Comment:
Can we please change this to INFO? Users may wish to ignore this message.

Reply:
Yeah, sure! Should I also change the error statement at line 37 (the one that says `"Buffer for file " .. notify_node .. " is modified"`), or just this one?

Reply:
I think just this one. Deleting a modified buffer truly is an error.

Reply:
Ok, done!